### PR TITLE
community contribution for issue #295.

### LINKED ISSUE #295: V4S: custom fields that contain "other" or "mailing" don't get copied to new contacts
When using the Sites pages, if the Contact has custom fields that contain "other" or "mailing" in their developer name, the V4S code in CreateOrUpdateContactFS() incorrectly assume the field is part of an address compound field, and it doesn't copy the field value to the new contact we are going to create.

### PR DESCRIPTION
**Pull request by davidhabib**
if you view the commit's diff with ?w=1 to ignore whitespace, you will see the simple changes that were made.  The reason the diff is showing the entire file changed, is that the CRLF are  being changed to just LF, which is our standard in Salesforce.org

# Critical Changes

# Changes
- Thank you community member Adam Studdard for providing the fixed code for an issue when the organization has custom fields with the word 'mailing' or 'other' in them.

# Issues Closed
Fixes #295 

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by davidhabib** [comment]
@kofijohnson  This pull request is ready for your review. if you view the commit's diff with ?w=1 to ignore whitespace, you will see the simple changes that were made.  The reason the diff is showing the entire file changed, is that the CRLF are  being changed to just LF, which is our standard in Salesforce.org

File: src/classes/VOL_SharedCode.cls [hunk 1]
```diff
@@ -605,7 +605,23 @@ global with sharing class VOL_SharedCode {
             contact.Volunteer_Notes__c = null;
             return conNew.Id;
         }       
-    }    
+    }
+
+    /*******************************************************************************************************
+    * @description detects standard mailing address fields
+    * @return Boolean True if it is a standard mailing address field, false if not.
```
Comment (kofijohnson) [inline src/classes/VOL_SharedCode.cls]: @davidhabib, minor fix here, could you add the @param to the Apex Doc.

File: src/classes/VOL_SharedCode.cls [hunk 2]
```diff
@@ -605,7 +605,23 @@ global with sharing class VOL_SharedCode {
...
+    /*******************************************************************************************************
+    * @description detects standard mailing address fields
+    * @return Boolean True if it is a standard mailing address field, false if not.
+    */
+    private static Boolean isMailingAddressField(String strField) {
+        return (strField.containsIgnoreCase('mailing') && !strField.endsWith('__c'));
+    }
+
+    /*******************************************************************************************************
+    * @description detects standard other address fields
```
Comment (kofijohnson) [inline src/classes/VOL_SharedCode.cls]: Same here, @param to the Apex Doc.

**Comment by davidhabib** [comment]
@kofijohnson I've updated the pull request the changes you requested.  thanks.

**Pull request merged**